element_lab/lab.py

```diff
@@ -79,59 +104,13 @@ class ProtocolType(dj.Lookup):
 class Protocol(dj.Lookup):
     definition = """
     # protocol approved by some institutions like IACUC, IRB
-    protocol                : varchar(16)
+    protocol                : varchar(36)
     ---
     -> ProtocolType
     protocol_description='' : varchar(255)
     """
 
 
-@schema
```

Comment:
Moving this will break the [nwb export](https://github.com/datajoint/element-lab/blob/main/element_lab/export/nwb.py). Should this PR also update schema name there?

Reply:
Good catch @CBroz1.  Yes, let's update `element_lab.export.nwb` in this pull request.